spec/controllers/submissions_controller_spec.rb

```diff
@@ -1847,13 +1847,14 @@
           assignment.update!(url_submit: true)
         end
 
-        it 'should return the file type' do
+        it 'should return the file type and non-zero size' do
```

Comment:
These test updates are good, but you should also add some for when the `max_size` parameter is passed.